index.html

```diff
@@ -3,7 +3,9 @@
   <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
   <title>Looking for a Needle in a haystack</title>
   <link rel="stylesheet" href="reset.css">
-  <link rel="stylesheet" type="text/css" media="all" href="style.css">
+  <link rel="stylesheet" href="style.css">
```

Comment:
What is the reasoning behind splitting these into two files? Do you know of any disadvantages of doing this?